db/migrate/20221026194149_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] The associations between models must be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Which means 🤔 The Post belongs-to user through the `'author_id'` column of the Post table. Now, we need to show that, the  `'author_id'` reference the User class modal. Kindly write the query like this so the foreign key name will be `author_id`:

```ruby
add_reference :posts, :author, null: false, foreign_key: { to_table: :users }
```

Reply:
@BenMukebo ,
Thank you for reviewing my project. But, as I can see I have already set the foreign key. I used the following syntax:
 rails generate AddUserRefToPost user:references and It generated the following output add_reference :posts, :user, null: false, foreign_key: true. Can you please tell me the syntax of how to do it the way you suggested?

Reply:
Hello @Ney2, We need to folow the ERD and according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). 👇🏻 

![image](https://user-images.githubusercontent.com/79058364/198257147-e2e36fe5-3cd1-4902-97b7-db872a5f213c.png)


They use 'author_id'  column as the foreign key reference the User class and not `users` column that is why we need to create our migration in this way 
eg: ```add_reference :posts, :author, null: false, foreign_key: { to_table: :users }```

Reply:
Thank you for the explanation.

Reply:
You're welcome 🙏🏻